### DISCUSSION TITLE
[Unstable]Interacting with passenger in raptorescort misson leads to mission failed.

### DISCUSSION THREAD

**Discussion by Cyana-ir** [FakeFishGames/Barotrauma#14707] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

After receive Civilian Transport mission unlocked by "man and his raptor" event, (mission id: raptorescort ) the mission will failed if you interacts witsh the master of the raptor, who even lives at the end of the round.
After testing, if you ignore the event's exclamation mark, then mission will be  successed correctly.

So，may be there are some problems in ScriptedEvent "manandhisraptor2" , I guess?
thanks for https://steamcommunity.com/profiles/76561198335732003/. But he doesn't know how to report, so here I am.

### Reproduction steps

1. start a singleplayer campaign.
2. enablecheats
3. unlockmission raptorescort
4. start the round
5. teleportsub end and dock.
  5.1 talk to passenger before docking, and the mission failed.
  5.2 straightly dock, and the mission successed.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

Other

### -

V1.6.9.0(unstable)

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell

```

**Comment by NilanthAnimosus** [comment]
Thank you for the report! Should be fixed now in a future build (Likely after PvP overhaul and ruin's improvements update)
